osclib/accept_command.py

```diff
@@ -78,11 +125,15 @@ def perform(self, project, force=False):
             oldspecs = self.api.get_filelist_for_package(pkgname=req['package'],
                                                          project=self.api.project,
                                                          extension='spec')
-            change_request_state(self.api.apiurl,
-                                 str(req['id']),
-                                 'accepted',
-                                 message='Accept to %s' % self.api.project)
-            self.create_new_links(self.api.project, req['package'], oldspecs)
+            if 'type' in req and req['type'] == 'delete' and self.api.delreq_review:
+                # Virtually accept to the delete request
```

Comment:
nit: `s/to //`

Although really it now says also exactly the same thing the method name does.